osi_lane.proto

```diff
@@ -817,6 +816,45 @@ message LaneBoundary
         // See \c LaneBoundary .
         //
         optional double height = 3;
+
+        // Alternation of dashes in case of a dashed lane boundary. In
+        // context, this field gives information about the location of
+        // dashes on the boundary line.
+        //
+        optional Dash dash = 4;
+
+        // This enum describes the alternation of dashes in case of a
+        // dashed lane boundary.
+        //
+        // \note The enum descriptions adhere to the definition direction
+        // of the lane boundary points, meaning that start or end of a dash
+        // are understood with respect to the direction in which the points
+        // of the boundary line are defined.
+        //
+        enum Dash
+        {
+            // The current state of the dash alternation is not known (must
+            // not be used in ground truth). 
+            //
+            DASH_UNKNOWN = 0;
+
+            // Other (unspecified but known) type of dash alternation state. 
```

Comment:
Another type of dash alternation state that is unspecified but known.

Reply:
See above, if we improve the wording of _OTHER and _UNKNOWN, which we should, we should do this in one fell swoop across the repo, not as part of this PR (leading to inconsistent wording across enums).